packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -171,20 +171,33 @@ export type VideoPlayerStatus = 'idle' | 'loading' | 'readyToPlay' | 'error';
 
 export type VideoSource =
   | string
+  | number
   | {
       /**
        * The URI of the video.
+       *
+       * This property is exclusive with the `assetId` property, when both are present the `assetId` will be ignored.
        */
-      uri: string;
+      uri?: string;
+
+      /**
+       * The assetId of a local video asset, acquired with the `require` function.
```

Comment:
```suggestion
       * The asset ID of a local video asset, acquired with the `require` function.
```

This is the explanation of the property with the same name. We need to define it clearly by considering the asset as the main subject being talked about and not referencing the property name.